samples/dotnet/kernel-syntax-examples/Example22_b_OpenApiSkill_Jira.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Connectors.WebApi.Rest;
+using Microsoft.SemanticKernel.Orchestration;
+using Microsoft.SemanticKernel.Skills.OpenAPI.Authentication;
+using Microsoft.SemanticKernel.Skills.OpenAPI.Skills;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using RepoUtils;
+
+public static class Example22_b_OpenApiSkill_Jira
+{
+    public static async Task RunAsync()
+    {
+        var kernel = new KernelBuilder().WithLogger(ConsoleLogger.Log).Build();
+        var contextVariables = new ContextVariables();
+
+        string s_serverURL = "https://<jiraProject>.atlassian.net/rest/api/latest/";
+        contextVariables.Set("server-url", s_serverURL);
+
+        IDictionary<string, ISKFunction> jiraSkills;
+        var tokenProvider = new BasicAuthenticationProvider(AuthenticateWithBasicApiTokenAsync);
```

Comment:
Could we inline the callback for readability? We want to make it easy for developers to see the whole flow at once.